crates/invoker-impl/Cargo.toml

```diff
@@ -30,14 +30,17 @@ bytes = { workspace = true }
 bytestring = { workspace = true }
 codederror = { workspace = true }
 futures = { workspace = true }
+gardal = { workspace = true , features = ["async", "tokio-hrtime"]}
```

Comment:
To avoid taking new dependencies, let's remove tokio-hrtime for now and only add if we have evidence that it's needed in our case.